### PR TITLE
Add code to allow SAP create with a placement group

### PR DESCRIPTION
**Pull request by rmoralesjr**
Add code to allow SAP create with a placement group
and include document statements with the restrictions
that apply with placement groups on instance deployment

### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates OR Closes #0000

Output from acceptance testing:

=== RUN   TestAccIBMPIPlacementGroupBasic
--- PASS: TestAccIBMPIPlacementGroupBasic (2150.42s)
PASS

### CODE REVIEW

File: website/docs/r/pi_instance.html.markdown [hunk 1]
```diff
@@ -78,13 +78,13 @@ Review the argument references that you can specify for your resource.
...
-- `pi_placement_group_id`- (Optional, String) The ID of the placement group that the instance is in if any or empty quotes `""` to also indicate it is not in a placement group.
+- `pi_placement_group_id` - (Optional, String) The ID of the placement group that the instance is in or empty quotes `""` to indicate it is not in a placement group. The meta-argument `count` and a `pi_replicant` cannot be used when specifying a placement group ID. Instances provisioning in the same placement group must be provisioned one at a time; however, to provision multiple instances on the same host or different hosts then use `pi_replicants` and `pi_replication_policy` instead of `pi_placement_group_id`.
```
Comment (yussufsh) [inline website/docs/r/pi_instance.html.markdown:81]: `pi_replicant`-->`pi_replicants`

Comment (rmoralesjr) [reply website/docs/r/pi_instance.html.markdown:81]: Done

**Review by ChaseRun — APPROVED** [review]

File: ibm/service/power/resource_ibm_pi_instance.go [hunk 2]
```diff
@@ -220,9 +220,10 @@ func ResourceIBMPIInstance() *schema.Resource {
 				Description: "PIN Policy of the Instance",
 			},
 			helpers.PIInstanceImageId: {
-				Type:        schema.TypeString,
-				Required:    true,
-				Description: "PI instance image id",
+				Type:             schema.TypeString,
+				Required:         true,
+				Description:      "PI instance image id",
+				DiffSuppressFunc: flex.ApplyOnce,
```
Comment (hkantare) [inline ibm/service/power/resource_ibm_pi_instance.go:226]: Why is the diff suprress func applied ?

Comment (yussufsh) [reply ibm/service/power/resource_ibm_pi_instance.go:226]: @hkantare when using a stock image the result image id from the pvm instance depends on the storage controller used to deploy the image. Here user provided configuration is not always same as the instance image id. Terraform would always show a diff to be applied which was also failing the test cases at some places depending on the storage controller selected automatically by the service provider.

**Pull request merged**